src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> resultMap;
+        try {
+            File file = new File(fromFileName);
+            List<String> lines = Files.readAllLines(file.toPath());
+            resultMap = getResultMap(lines);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            for (String key : resultMap.keySet()) {
+                writer.write(key);
+                writer.write(",");
+                writer.write(resultMap.get(key));
```

Comment:
You are writing the value of `resultMap.get(key)` directly, which is an `Integer`. You need to convert it to a `String` using `String.valueOf(resultMap.get(key))` or `resultMap.get(key).toString()` to avoid a compilation error.